public/js/pimcore/object/tree.js

```diff
@@ -276,7 +276,9 @@ pimcore.registerNS("pimcore.object.tree");
                      this.onTreeNodeMove(record, record.parentNode, overModel, 0);
                  }
              }.bind(this));
+         }
 
+         if (typeof this.treeNodeMoveParameter.oldParent.getOwnerTree !== "function") {
```

Comment:
```suggestion
         // recheck wheter the getOwnerTree is a function after reassignment onTreeNodeMove()
         if (typeof this.treeNodeMoveParameter.oldParent.getOwnerTree !== "function") {
```
it should be helping to spot that the getOwnerTree could be changed by `onTreeNodeMove()`
https://github.com/pimcore/admin-ui-classic-bundle/pull/757/files#diff-3b41334b430020814d1a30721473a6d8647e88045cb572be36931f80d28522f2R263